app/views/drawings/_form.html.haml

```diff
@@ -1,53 +1,48 @@
-.container
-  %h3
+.col-xs-12.text-center
+  %h1.add-margin.top.bottom
     = form_title
-  .row.drawings-wrapper
-    = simple_form_for @drawing, html: { multipart: true } do |f|
-      .col-md-5.col-sm-12.col-xs-12
+.row.drawings-wrapper.no-padding
+  .col-xs-12.col-sm-11.position.central.no-padding
+    = simple_form_for @drawing, html: {multipart: true } do |f|
+      .col-xs-12.col-md-4
         .image-wrap.text-center
-          %h4 Upload an image
-          %i (this is required)
           = image_tag preview_image(@drawing), id: 'image-preview', class: 'img-responsive'
-          %br
-          %b Allowed image types: jpg/jpeg, png, tiff, gif, bmp
-          = f.input :image, label: false, required: false, input_html: { onChange: 'loadFile(event)' }
-        .form-group.form-consent
-          = f.input :image_consent, label: @drawing.image_consent_text, input_html: { checked: image_consent_default }
+          %b Allowed image types: jpg, jpeg, png, tiff, gif, bmp
+          =f.input :image, label: false, required: false, input_html: { onChange: 'loadFile(event)' }
 
-      .col-md-7.col-sm-12.col-xs-12
-        %h4 Details of Drawing (required when marked complete)
-        .form-group
-          = f.input :country, priority: TOP_COUNTRIES, selected: (@drawing.new_record? ? current_user.country : @drawing.country), label: "Country drawn in", include_blank: "-- select country that picture was drawn in --", class: 'form-control'
-        .form-group
-          = f.input :subject_matter, collection: ["Home / Country of origin", "Transit", "Camp life", "Future hopes / destination"], label: 'Subject matter of drawing', required: false, prompt: "-- select an option --"
-        .form-group
-          = f.input :description, label: 'Description of drawing', placeholder: 'e.g. Children playing in the garden, mountains in background'
-        .form-group
-          = f.input :mood_rating, as: :select, collection: mood_select_box, label: 'How would you rate the mood of the drawing?', prompt: "-- scale is from 1 (sad/tragic) to 5 (happy/joyful) --"
+      .forms-wrapper.col-xs.12.col-sm-8.no-padding
+        .col-xs-12.col-md-6
+          .position.central.form-section
+            .form-title
+              %h2 Drawing Details
+            .form-group
+              = f.input :country, priority: TOP_COUNTRIES, selected: (@drawing.new_record? ? current_user.country : @drawing.country), label: "Country drawn in", class: 'form-control'
+              .form-group
+              = f.input :subject_matter, collection: Drawing.subject_matters, label: "Subject matter of drawing", required: false
+            .form-group
+              = f.input :mood_rating, as: :radio_buttons, collection: mood_select_box, label: 'How would you rate the mood of the drawing?'
+            .form-group
+              = f.input :description, as: :text, input_html: {:rows => 8 }, label: 'Description of drawing'
```

Comment:
Small syntax tip: standard for hash `:symbol_key => value` references can now be `symbol_key: value`